frontend/src/scenes/project/Settings/DataAttributes.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Input, Skeleton } from 'antd'
+import { useActions, useValues } from 'kea'
+import React, { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+export function DataAttributes(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const { patchCurrentTeam } = useActions(teamLogic)
+    const [value, setValue] = useState(currentTeam?.data_attributes?.join(', ') || '')
+
+    if (!currentTeam) {
+        return <Skeleton paragraph={{ rows: 0 }} active />
+    }
+
+    return (
+        <>
+            <p>
+                Specify a comma-separated list of{' '}
+                <a
+                    href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes"
+                    rel="noreferrer noopener"
+                >
+                    data attributes
+                </a>{' '}
+                used in your app. For example: <code>data-attr, data-custom-id, data-myref</code>. These attributes will
+                be used when creating actions to match unique elements on your pages.
+            </p>
+            <p>
+                For example, when creating an action on your CTA button, the best selector could be something like:{' '}
+                <code>div &gt; form &gt; button:nth-child(2)</code>. However all buttons in your app have a{' '}
+                <code>data-custom-id</code> attribute. If you whitelist it here, the selector for your button will
+                instead be <code>button[data-custom-id='cta-button']</code>.
+            </p>
+            <div>
+                <Input
```

Comment:
WDYT about using SelectGradientOverflow here to do the multi-input rather than doing this comma-based? See https://github.com/PostHog/posthog/pull/3756/files#diff-f180e14d42a7636d030be25ac92466e7a890170edfe246d5b9f0f63c46121634R80 for an example.

Reply:
Good idea! Though since we're not autocompleting anything here, I just used `Select` instead of `SelectGradientOverflow`.